### ISSUE TITLE
Slider with no title (feature request)

### ISSUE THREAD

**Issue by gryBox** [bokeh/bokeh#5175]
Currently current bokeh slider always displays a value.  Even when setting the `title` to empty

```
from bokeh.io import output_file, show
from bokeh.layouts import widgetbox
from bokeh.models.widgets import Slider

output_file("slider.html")

slider = Slider(start=0, end=10, value=1, step=.1, title="")

show(widgetbox(slider))
```
![image](https://cloud.githubusercontent.com/assets/16763201/18595811/f24475e8-7c0b-11e6-8282-744885bd0ca0.png)

There are instances when a user does not want to display any value:   One use case is when "grouping" two sliders and displaying a custom title value in some other widget e.g. textbox

**Comment by mattpap** [comment]
@gryBox, I implemented requested behavior in PR #5178, just the syntax for no text at all will be `title=None`.

**Comment by gryBox** [comment]
@mattpap  Thank you! ..
